### PR TITLE
FOSDEM 2016

### PR DESCRIPTION
**Pull request by dentarg**
Summary of the talks I saw at FOSDEM.

### CODE REVIEW

**Comment by dentarg** [comment]
@walro has been granted permission (or is being forced to?) to correct typos :smile:

**Comment by walro** [comment]
lolz, should've kept quiet!

File: _posts/2016-02-04-fosdem-2016.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: FOSDEM 2016
+author: patrik
+categories: open-source conference
+---
+
+For the last weekend of January, I traveled with Johan to Brussels to attend the two-day open source event [FOSDEM 2016](https://fosdem.org/2016/). This was our second time going to the conference, as we also visited it last year. I thought I should share notes from some of the talks I listened to, so here we go.
```
Comment (walro) [inline _posts/2016-02-04-fosdem-2016.md]: Perhaps link Johan's GitHub profile as I imagine not everyone knows who Johan is? Or add "my colleague"

Comment (dentarg) [inline _posts/2016-02-04-fosdem-2016.md]: Sure

File: _posts/2016-02-04-fosdem-2016.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+#### [Single-pass Graph Streaming Analytics with Apache Flink](https://fosdem.org/2016/schedule/event/graph_processing_apache_flink/)
+
+This talk, done by the PhD students [Vasia Kalavri](https://github.com/vasia) and [Paris Carbone](https://github.com/senorcarbone) from [Royal Institute of Technology](https://www.kth.se/en), was probably way over my head, so I really can't say much about it.
+
+The gist of the talk, I think, was that you can do a lot of things with just a "summary" of your complete graph. With [Gelly Streaming](https://github.com/vasia/gelly-streaming), Vasia and Paris have implemented an API that allows you to use this technique together with data streaming platform [Apache Flink](https://flink.apache.org/).
+
+#### [Testing complex software in CI](https://fosdem.org/2016/schedule/event/testing_complex_software_in_ci/)
+
+This talk was held by two guys working for Red Hat. One of them is working on [Samba](https://www.samba.org/) and the other one is working on [SSSD](https://fedorahosted.org/sssd/). Both also works on [libssh](https://www.libssh.org/).
```
Comment (walro) [inline _posts/2016-02-04-fosdem-2016.md]: works on -> work on

Comment (dentarg) [inline _posts/2016-02-04-fosdem-2016.md]: You can just go ahead and correct all typos if you like :)

**Pull request merged**